### PR TITLE
Vendor in containers/buildah@v1.26.1

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Review by lsm5 — APPROVED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
@rhatdan couple of tests failed, I don't think they're flakes.

```
[+0025s] not ok 160 [10-images] GET images/json?filters={"label":["xyz"]} : length
[+0025s] #  expected: 1
[+0025s] #    actual: 0

[+0025s] not ok 162 [10-images] GET libpod/images/json?filters={"label":["xyz"]} : length
[+0025s] #  expected: 1
[+0025s] #    actual: 0
```

File: test/e2e/images_test.go [hunk 1]
```diff
@@ -422,7 +422,7 @@ LABEL "com.example.vendor"="Example Vendor"
 		Expect(result.OutputToStringArray()).To(Not(Equal(result1.OutputToStringArray())))
 	})
 
-	It("podman image prune --filter", func() {
+	FIt("podman image prune --filter", func() {
```
Comment (flouthoc) [inline test/e2e/images_test.go:425]: @rhatdan Could you please drop changes in `test/e2e/images_test.go` this is not needed and I think this is causing CI to fail.

File: test/e2e/images_test.go [hunk 2]
```diff
@@ -435,7 +435,7 @@ RUN > file2
 		result := podmanTest.Podman([]string{"image", "prune", "--filter", "label=abc", "--force"})
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(0))
-		Expect(result.OutputToStringArray()).To(HaveLen(1))
+		Expect(result.OutputToString()).To(BeEmpty())
```
Comment (flouthoc) [inline test/e2e/images_test.go:438]: This is the change which is causing issues `test/e2e/images_test.go` this should be reverted to `Expect(result.OutputToStringArray()).To(HaveLen(1))`

File: test/buildah-bud/apply-podman-deltas [hunk 3]
```diff
@@ -188,6 +201,15 @@ skip_if_remote "local keyfile not sent to podman-remote" \
...
 
+# https://github.com/containers/buildah/pull/3823
+# If this is possible with podman-remote, it'll take way more Go skills
+# to implement than what Ed can do.
+skip_if_remote "--output option not implemented in podman-remote" \
+               "build with custom build output and output rootfs to directory" \
+               "build with custom build output and output rootfs to tar" \
+               "build with custom build output and output rootfs to tar by pipe" \
+               "build with custom build output must fail for bad input"
+
```
Comment (Luap99) [inline test/buildah-bud/apply-podman-deltas:212]: If podman-remote does not support this, then the code should error when the option is used.
Second it has to be documented in the --output option that this is not support via remote.

Comment (flouthoc) [reply test/buildah-bud/apply-podman-deltas:212]: @Luap99 Yes I'll open a followup after this vendor to address this.

**Pull request merged**